x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/utils/connector_status_helpers.ts

```diff
@@ -87,9 +88,10 @@ export function connectorStatusToColor(connector: Connector): 'warning' | 'dange
   if (
     isLastSeenOld(connector) ||
     connectorStatus === ConnectorStatus.ERROR ||
-    connector.error === SyncStatus.ERROR ||
```

Comment:
The previous check didn't make sense (error can be arbitrary string), I adapted to check for last sync status